test-suite/js/src/test/scala/org/scalajs/testsuite/utils/HTMLRunner.scala

```diff
@@ -127,6 +140,7 @@ object HTMLRunner extends js.JSApp {
       def appendChild(child: Element): Unit = js.native
       def setAttribute(name: String, value: String): Unit = js.native
       var textContent: String = js.native
+      val scrollHeight: Int = js.native
```

Comment:
This should be a `def`, not a `val`, as its value can definitely change over time.